src/pages/trade/ui/order-form/store/index.ts

```diff
@@ -57,7 +57,6 @@ class OrderFormStore {
     makeAutoObservable(this);
 
     void this.calculateGasFee();
-    void this.calculateExchangeRate();
```

Comment:
@JasonMHasperhoven can you validate this is ok? On load, there is no assetIds set so it is throwing. Upon the assetIds loading in the store, it re-renders and runs the calculations later anyway.

Reply:
on that note, why are we even running `void this.calculateGasFee()` in the constructor? It similarly would be undefined

Reply:
Yeah, we kinda have a race condition with these. I think we can probably delete both.